src/index.html

```diff
@@ -3,11 +3,62 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width,
+          user-scalable=no,
+          initial-scale=1.0,
+          maximum-scale=1.0,
+          minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com"
+    >
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet"
+    >
+    <link rel="stylesheet"
+          href="./styles/index.scss"
+    >
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+
+      <img
+        src="images/imac.jpeg"
+        alt="iMac computer image"
+        class="card-image"
+      >
+
+      <h2 class="card-main-text">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card-subtitle-text">Product code: 195434</p>
+
+      <div class="card-stars"
+           data-qa="card"
+      >
```

Comment:
![Screenshot from 2023-07-10 19-45-08](https://github.com/mate-academy/layout_product-cards/assets/86923089/a0dd4f43-79ed-4d40-91b2-bbec95ad4437)

Fix it in this way (everywhere):
```suggestion
      <div class="card-stars" data-qa="card">
```
write attributes in a row if there are less than 2 attributes

Reply:
Done, thanks for explanation